sunpy/map/header_helper.py

```diff
@@ -52,6 +55,13 @@ def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None,
         Pixel scaling along x and y axis (i.e. the spatial scale of the pixels (dx, dy)). These are
         expected to be Cartestian ordered, i.e [dx, dy].
         Defaults to ``([1., 1.] arcsec/pixel)``.
+    rotation_angle : `~astropy.unit.Quantity`, optional
+        Coordinate system rotation angle, will be converted to a rotation
+        matrix and stored in the `PCi_j` matrix. Can not be specified with
```

Comment:
```suggestion
        matrix and stored in the "PCi_j" matrix. Can not be specified with
```
Maybe?

Reply:
I think it will read better in verbatim, but needs to be double back tick?

Reply:
Either works for me.